docs/conf.md

```diff
@@ -0,0 +1,64 @@
+# rambo.conf
+
+The `rambo.conf` file is required at the top level in your project directory. It is an INI config file that can specify options on various commands. Options passed to the CLI will take precedence over options set via this config file. If you're repeating the same CLI options, setting those options in this config might make your life a little easier. Further, if you intend on tracking your Rambo project in version control, it can be very handy to set some options in this config that match the purpose of your project.
+
+Options on the base `rambo` command and the subcommand `up` can be set in `rambo.conf`. The INI section refers to the command used in the CLI, and the keys/values within that section refers to the options being set on that command. For example, a useful `rambo.conf` could look like this:
+
+```ini
+[up]
+provider = digitalocean
+guest_os = centos-7
+```
+
+which is equivalent to:
+
+```bash
+rambo up --provider digitalocean --guest-os centos-7
+```
+
+Setting the config file to this allows you to type simply `rambo up` to run `up` with the `provider` and `guest-os` options set in the rambo.conf, and not specified in the CLI.
+
+## Sections
+
+There are only two sections for the conf file that Rambo uses: `[base]` and `[up]`. These correspond to the base rambo command / entry point, and to the `up` command. We expect that you will most often only need to set options in the `[up]` section for the `up` command like `drive-size`, and only rarely need to set options in the `[base]` seection for the `rambo` entry point options like `vagrant-dotfile-path`.
```

Comment:
I'm not sure that this means: These correspond to the base rambo command / entry point, and

Reply:
How about, "These correspond to the options directly on the base `rambo` command, and on the `up` subcommand."?

Or option 2), "These correspond to the options directly on the `rambo` command, and on the `up` subcommand.", together with actually changing the sections to being `[rambo]` and `[up]`. That would mean actually changing 3 lines of code to use `Rambo` instead of `Base`, which I'm ok with, and maybe give a better user experience with the conf file.

Reply:
You've got to keep in mind the click vocabulary means nothing to a user. It is not helpful.

Reply:
And I'm not seeing what reason there is to change how the config file is working.